### PR TITLE
Merge solver.xsd and benchmark.xsd

### PR DESCRIPTION
**Pull request by rsynek**
An alternative to having a single namespace: merging solver.xsd into benchmark.xsd, so that:
- optaplanner-core.jar can contain solver.xsd under its namespace
- optaplanner-benchmark.jar can contain benchmark.xsd under its own namespace without importing the solver.xsd (a thus without the need of using prefixes in the benchmarkConfig.xml)

Related PR:
https://github.com/kiegroup/optaplanner-website/pull/271

### CODE REVIEW

**Review by triceo — APPROVED** [review]
The best thing possible, given the circumstances.
Question: don't all the Optaplanner Examples configs need updating too? I remember seeing a PR earlier, changing them to use the XSD which now changes location.

**Comment by rsynek** [comment]
> The best thing possible, given the circumstances.
> Question: don't all the Optaplanner Examples configs need updating too? I remember seeing a PR earlier, changing them to use the XSD which now changes location.

Sure they do; I wanted to have some feedback first before I change all of them. Also, similar cleanup of the JAR files as in https://github.com/kiegroup/optaplanner/pull/908 will be needed too (we want to keep the benchmark.xsd, but remove the benchmark-with-import.xsd).

File: optaplanner-examples/src/main/resources/org/optaplanner/examples/cheaptime/solver/cheapTimeSolverConfig.xml [hunk 1]
```diff
@@ -14,7 +15,7 @@
     <!--<scoreDrl>org/optaplanner/examples/cheaptime/solver/cheapTimeConstraints.drl</scoreDrl>-->
     <initializingScoreTrend>ONLY_DOWN</initializingScoreTrend>
     <!--<assertionScoreDirectorFactory>-->
-      <!--<easyScoreCalculatorClass>org.optaplanner.examples.cheaptime.solver.score.CheapTimeEasyScoreCalculator</easyScoreCalculatorClass>-->
```
Comment (triceo) [inline optaplanner-examples/src/main/resources/org/optaplanner/examples/cheaptime/solver/cheapTimeSolverConfig.xml:17]: Unnecessary change. This PR actually has plenty of them.
(Although does it matter? I leave it up to you, @rsynek .)

Comment (rsynek) [reply optaplanner-examples/src/main/resources/org/optaplanner/examples/cheaptime/solver/cheapTimeSolverConfig.xml]: Good catch, thanks.

**Comment by ge0ffrey** [comment]
Interesting, let me look at this tomorrow, after OptaPlanner Week event.
